0-EnglishVersion/README.md

```diff
@@ -1,6 +1,6 @@
 # <font size=5>Microsoft AI Education Community</font>
 
-This community is an AI education and learning community created by the AI-Edu team of Microsoft Research Asia (MSRA). We provide a series of AI tutorials, study cases, tools and other resources. Welcome to enjoy your AI trip in this community!
```

Comment:
This did not make sense.
I suggest: We hope you enjoy your AI learning experience in this community!